src/main/java/uk/gov/companieshouse/pscstatementdataapi/controller/PscStatementController.java

```diff
@@ -25,15 +29,15 @@ public class PscStatementController {
     @Autowired
     private PscStatementService pscStatementService;
 
-    @GetMapping("/company/{company_number}/persons-with-significant-control-statement/{statement_id}")
+    @GetMapping("/company/{company_number}/persons-with-significant-control-statements/{statement_id}")
```

Comment:
A lot of these mappings are very similar, is it worth looking at having an @Mapping at the controller level and appending what's needed for the get, put, delete etc?